diff --git a/sources/shiboken6/ApiExtractor/clangparser/compilerSupport.cpp b/sources/shiboken6/ApiExtractor/clangparser/compilerSupport.cpp
index 996968f..a072dfd 100644
--- a/sources/shiboken6/ApiExtractor/clangparser/compilersupport.cpp
+++ b/sources/shiboken6/ApiExtractor/clangparser/compilersupport.cpp
@@ -272,7 +272,7 @@
 {
     for (const char *envVar : {"LLVM_INSTALL_DIR", "CLANG_INSTALL_DIR"}) {
         if (qEnvironmentVariableIsSet(envVar)) {
-            const QString path = QFile::decodeName(qgetenv(envVar)) + u"/lib"_s;
+            const QString path = QFile::decodeName(qgetenv(envVar)) + u"/lib64"_s;
             if (QFileInfo::exists(path))
                 return path;
             qCWarning(lcShiboken, "%s: %s as pointed to by %s does not exist.",

diff --git a/sources/shiboken6/cmake/ShibokenHelpers.cmake b/sources/shiboken6/cmake/ShibokenHelpers.cmake
index 996968f..a072dfd 100644
--- a/sources/shiboken6/cmake/ShibokenHelpers.cmake
+++ b/sources/shiboken6/cmake/ShibokenHelpers.cmake
@@ -873,7 +873,7 @@
         set(libclang_directory_suffix "bin")
         set(libclang_suffix ".dll")
     else()
-        set(libclang_directory_suffix "lib")
+        set(libclang_directory_suffix "lib64")
         if(CMAKE_HOST_APPLE)
             set(libclang_suffix ".dylib")
         else()
