--- a/build_scripts/build_info_collector.py
+++ b/build_scripts/build_info_collector.py
@@ -66,7 +66,7 @@
                          py_include_dir):
     """Helper for finding the Python library on UNIX"""
     if py_libdir is None or not Path(py_libdir).exists():
-        py_libdir = Path(py_prefix) / "lib"
+        py_libdir = Path(py_prefix) / "lib64"
     if py_include_dir is None or not Path(py_include_dir).exists():
         directory = f"include/python{py_version}"
         py_include_dir = Path(py_prefix) / directory
@@ -203,6 +203,10 @@
             else:
                 py_scripts_dir = py_prefix / "bin"
             self.py_scripts_dir = py_scripts_dir
+            # ASWF: with a non relocatable Python Conan package, sysconfig
+            # returns a path to the Python package inside the build folder
+            directory = f"include/python{py_version}"
+            py_include_dir = py_prefix / directory
         else:
             # We don't look for an interpreter when cross-compiling.
             py_executable = None
