--- a/build_scripts/build_info_collector.py
+++ b/build_scripts/build_info_collector.py
@@ -203,6 +203,10 @@
             else:
                 py_scripts_dir = py_prefix / "bin"
             self.py_scripts_dir = py_scripts_dir
+            # ASWF: with a non relocatable Python Conan package, sysconfig
+            # returns a path to the Python package inside the build folder
+            directory = f"include/python{py_version}"
+            py_include_dir = py_prefix / directory
         else:
             # We don't look for an interpreter when cross-compiling.
             py_executable = None
