diff --git cmake/defaults/Packages.cmake cmake/defaults/Packages.cmake
index d99dd79ac..bfda2778a 100644
--- cmake/defaults/Packages.cmake
+++ cmake/defaults/Packages.cmake
@@ -266,6 +266,16 @@
     # --Ptex
     if (PXR_ENABLE_PTEX_SUPPORT)
         find_package(PTex REQUIRED)
+        get_target_property(
+            PTEX_INCLUDE_DIR
+            Ptex::Ptex_dynamic
+            INTERFACE_INCLUDE_DIRECTORIES
+        )
+        get_target_property(
+            PTEX_LIBRARY
+            Ptex::Ptex_dynamic
+            INTERFACE_LINK_LIBRARIES
+        )
         add_definitions(-DPXR_PTEX_SUPPORT_ENABLED)
     endif()
     # --OpenVDB
@@ -300,6 +310,9 @@

 if (PXR_BUILD_ALEMBIC_PLUGIN)
     find_package(Alembic REQUIRED)
+    set(ALEMBIC_FOUND TRUE)
+    set(ALEMBIC_INCLUDE_DIR ${Alembic_INCLUDE_DIR})
+    set(ALEMBIC_LIBRARIES ${Alembic_LIBRARIES})
     set(REQUIRES_Imath TRUE)
     if (PXR_ENABLE_HDF5_SUPPORT)
         find_package(HDF5 REQUIRED
--- pxr/usd/usdMtlx/CMakeLists.txt
+++ pxr/usd/usdMtlx/CMakeLists.txt
@@ -54,8 +54,8 @@
         usdShade
         usdUI
         usdUtils
-        MaterialXCore
-        MaterialXFormat
+        materialx::MaterialXCore
+        materialx::MaterialXFormat

     CPPFILES
         discovery.cpp
--- pxr/imaging/plugin/hioOiio/CMakeLists.txt
+++ pxr/imaging/plugin/hioOiio/CMakeLists.txt
@@ -22,11 +22,11 @@
         gf
         hio
         tf
-        ${OIIO_LIBRARIES}
+        ${OpenImageIO_LIBRARIES}
         ${__OIIO_IMATH_LIBS}

     INCLUDE_DIRS
-        ${OIIO_INCLUDE_DIRS}
+        ${OpenImageIO_INCLUDE_DIR}
         ${__OIIO_IMATH_INCLUDE}

     CPPFILES
--- pxr/imaging/hdx/CMakeLists.txt
+++ pxr/imaging/hdx/CMakeLists.txt
@@ -11,8 +11,8 @@

 set (optionalCppFiles "")
 if (PXR_BUILD_OPENCOLORIO_PLUGIN)
-    list(APPEND optionalLibs ${OCIO_LIBRARIES})
-    list(APPEND optionalIncludeDirs ${OCIO_INCLUDE_DIRS})
+    list(APPEND optionalLibs ${OpenColorIO_LIBRARIES})
+    list(APPEND optionalIncludeDirs ${OpenColorIO_INCLUDE_DIR})
 endif()

 pxr_library(hdx
--- pxr/usd/plugin/sdrOsl/CMakeLists.txt
+++ pxr/usd/plugin/sdrOsl/CMakeLists.txt
@@ -20,13 +20,13 @@
         vt
         ar
         sdr
-        ${OSL_QUERY_LIBRARY}
-        ${OIIO_LIBRARIES}
+        OSL::oslquery
+        ${OpenImageIO_LIBRARIES}
         ${__OSL_IMATH_LIBS}

     INCLUDE_DIRS
         ${OSL_INCLUDE_DIR}
-        ${OIIO_INCLUDE_DIRS}
+        ${OpenImageIO_INCLUDE_DIR}
         ${__OSL_IMATH_INCLUDE}

     PUBLIC_CLASSES
--- pxr/imaging/hioOpenVDB/CMakeLists.txt
+++ pxr/imaging/hioOpenVDB/CMakeLists.txt
@@ -28,10 +28,10 @@
         tf
         usd
         ${__VDB_IMATH_LIBS}
-        ${OPENVDB_LIBRARY}
+        ${OpenVDB_LIBRARIES}

     INCLUDE_DIRS
-        ${OPENVDB_INCLUDE_DIR}
+        ${OpenVDB_INCLUDE_DIR}
         ${Boost_INCLUDE_DIRS}

     PUBLIC_CLASSES
--- pxr/imaging/hdSt/CMakeLists.txt
+++ pxr/imaging/hdSt/CMakeLists.txt
@@ -15,16 +15,16 @@
 set(optionalPrivateClasses "")
 if (${PXR_ENABLE_MATERIALX_SUPPORT})
     list(APPEND optionalLibs
-        MaterialXGenShader
-        MaterialXRender
-        MaterialXCore
-        MaterialXFormat
-        MaterialXGenGlsl
+        materialx::MaterialXGenShader
+        materialx::MaterialXRender
+        materialx::MaterialXCore
+        materialx::MaterialXFormat
+        materialx::MaterialXGenGlsl
         hdMtlx
     )
     if (PXR_ENABLE_METAL_SUPPORT)
         list(APPEND optionalLibs
-            MaterialXGenMsl
+            materialx::MaterialXGenMsl
         )
     endif()
     list(APPEND optionalPrivateClasses
--- pxr/imaging/hdMtlx/CMakeLists.txt
+++ pxr/imaging/hdMtlx/CMakeLists.txt
@@ -11,8 +11,8 @@
         trace
         usdMtlx
         vt
-        MaterialXCore
-        MaterialXFormat
+        materialx::MaterialXCore
+        materialx::MaterialXFormat

     PUBLIC_CLASSES
         hdMtlx
--- pxr/pxrConfig.cmake.in
+++ pxr/pxrConfig.cmake.in
@@ -104,6 +104,45 @@
     find_dependency(Imath)
 endif()

+# If OpenColorIO support was enabled for this USD build, find the associated
+# import targets by invoking the same OpenColorIO cmake package config that
+# was used for that build. This can be overridden by specifying a different
+# OpenColorIO_DIR when running cmake.
+if(@PXR_BUILD_OPENCOLORIO_PLUGIN@)
+    if (NOT DEFINED OpenColorIO_DIR)
+        if (NOT [[@OpenColorIO_DIR@]] STREQUAL "")
+            set(OpenColorIO_DIR [[@OpenColorIO_DIR@]])
+        endif()
+    endif()
+    find_dependency(OpenColorIO)
+endif()
+
+# If OpenVDB support was enabled for this USD build, find the associated
+# import targets by invoking the same OpenVDB cmake package config that
+# was used for that build. This can be overridden by specifying a different
+# OpenVDB_DIR when running cmake.
+if(@PXR_ENABLE_OPENVDB_SUPPORT@)
+    if (NOT DEFINED OpenVDB_DIR)
+        if (NOT [[@OpenVDB_DIR@]] STREQUAL "")
+            set(OpenVDB_DIR [[@OpenVDB_DIR@]])
+        endif()
+    endif()
+    find_dependency(OpenVDB)
+endif()
+
+# If Ptex support was enabled for this USD build, find the associated
+# import targets by invoking the same PTex cmake package config that
+# was used for that build. This can be overridden by specifying a different
+# PTex_DIR when running cmake.
+if(@PXR_ENABLE_PTEX_SUPPORT@)
+    if (NOT DEFINED PTex_DIR)
+        if (NOT [[@PTex_DIR@]] STREQUAL "")
+            set(PTex_DIR [[@PTex_DIR@]])
+        endif()
+    endif()
+    find_dependency(PTex)
+endif()
+
 # If this build is using a custom work implementation, find the package
 # providing that implementation.
 if(NOT "@PXR_WORK_IMPL_PACKAGE@" STREQUAL "")
