diff --git a/CMakeLists.txt b/CMakeLists.txt
index 570e633..1338c2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,8 +121,12 @@ set(OpenGL_GL_PREFERENCE GLVND)
 find_package(Threads REQUIRED)
 find_package(GLUT REQUIRED)
 find_package(OpenGL REQUIRED)
-find_package(PythonInterp REQUIRED)
-find_package(PythonLibs REQUIRED)
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
+    find_package(Python COMPONENTS Interpreter Development)
+else()
+    find_package(PythonInterp REQUIRED)
+    find_package(PythonLibs REQUIRED)
+endif()
 find_package(ZLIB)
 if (ZLIB_FOUND)
     add_definitions(-DPARTIO_USE_ZLIB)
