diff --git a/src/py/CMakeLists.txt b/src/py/CMakeLists.txt
index 4d7c45b..56ac174 100644
--- a/src/py/CMakeLists.txt
+++ b/src/py/CMakeLists.txt
@@ -52,7 +52,11 @@ target_include_directories(
 )
 target_link_libraries(partio_mod PRIVATE ${PYTHON_LIBRARIES} partio)

-set(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages" )
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
+    set(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages" )
+else()
+    set(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages" )
+endif()

 install(TARGETS partio_mod DESTINATION ${PYTHON_DEST})
 install(FILES partio.i DESTINATION ${CMAKE_INSTALL_DATADIR}/swig)

diff --git a/src/tools/CMakeLists.txt b/src/tools/CMakeLists.txt
index 8aac9ad..372bbaa 100644
--- a/src/tools/CMakeLists.txt
+++ b/src/tools/CMakeLists.txt
@@ -67,5 +67,9 @@ install(PROGRAMS partedit.py DESTINATION ${CMAKE_INSTALL_BINDIR} RENAME partedit
 install(PROGRAMS partjson.py DESTINATION ${CMAKE_INSTALL_BINDIR} RENAME partjson)
 install(PROGRAMS partinspect.py DESTINATION ${CMAKE_INSTALL_BINDIR} RENAME partinspect)

-set(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages" )
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
+    set(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages" )
+else()
+    set(PYTHON_DEST "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages" )
+endif()
 install(FILES partedit.py partjson.py partinspect.py DESTINATION ${PYTHON_DEST})

diff --git a/src/tests/CMakeLists.txt b/src/tests/CMakeLists.txt
index b796ecb..855d09f 100644
--- a/src/tests/CMakeLists.txt
+++ b/src/tests/CMakeLists.txt
@@ -50,7 +50,11 @@ else()
     endforeach(item)

     foreach(item testpartjson testpartio)
-        add_test(NAME ${item} COMMAND ${PYTHON_EXECUTABLE} -B ${CMAKE_CURRENT_SOURCE_DIR}/${item}.py)
+        if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
+            add_test(NAME ${item} COMMAND ${Python_EXECUTABLE} -B ${CMAKE_CURRENT_SOURCE_DIR}/${item}.py)
+        else()
+            add_test(NAME ${item} COMMAND ${PYTHON_EXECUTABLE} -B ${CMAKE_CURRENT_SOURCE_DIR}/${item}.py)
+        endif()
         install(PROGRAMS ${item}.py DESTINATION ${CMAKE_INSTALL_PARTIO_TESTDIR} RENAME ${item})
     endforeach(item)
 endif()

