diff --git src/bindings/python/CMakeLists.txt src/bindings/python/CMakeLists.txt
index 0eaec4be..e711295c 100755
--- src/bindings/python/CMakeLists.txt
+++ src/bindings/python/CMakeLists.txt
@@ -214,9 +214,9 @@
 target_compile_definitions(PyOpenColorIO
 	PRIVATE
 		PYOCIO_NAME=PyOpenColorIO
-		PY_VERSION_MAJOR=${Python_VERSION_MAJOR}
-		PY_VERSION_MINOR=${Python_VERSION_MINOR}
-		PY_VERSION_PATCH=${Python_VERSION_PATCH}
+		PY_VERSION_MAJOR=${Python3_VERSION_MAJOR}
+		PY_VERSION_MINOR=${Python3_VERSION_MINOR}
+		PY_VERSION_PATCH=${Python3_VERSION_PATCH}
 )

 ###############################################################################
@@ -241,7 +241,7 @@
 if(WIN32)
     set(_Python_VARIANT_PATH "${CMAKE_INSTALL_LIBDIR}/site-packages")
 else()
-    set(_Python_VARIANT_PATH "${CMAKE_INSTALL_LIBDIR}/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages")
+    set(_Python_VARIANT_PATH "${CMAKE_INSTALL_LIBDIR}/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages")
 endif()

 # Create an internal global variable to access it in another scope but not publicly visible.
