diff --git CMakeLists.txt CMakeLists.txt
index d99dd79ac..bfda2778a 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -32,6 +32,7 @@
 option(OTIO_INSTALL_PYTHON_MODULES    "Install OTIO pure Python modules/files" ON)
 option(OTIO_INSTALL_COMMANDLINE_TOOLS "Install the OTIO command line tools" ON)
 option(OTIO_FIND_IMATH                "Find Imath using find_package" OFF)
+option(OTIO_FIND_PYBIND11             "Find pybind11 using find_package" OFF)
 option(OTIO_FIND_RAPIDJSON            "Find RapidJSON using find_package" OFF)
 set(OTIO_PYTHON_INSTALL_DIR "" CACHE STRING "Python installation dir (such as the site-packages dir)")

@@ -252,6 +253,17 @@
     include_directories("${PROJECT_SOURCE_DIR}/src/deps/Imath/src")
 endif()

+#----- pybind11
+if(OTIO_FIND_PYBIND11)
+    find_package(pybind11 REQUIRED)
+    if (pybind11_FOUND)
+        message(STATUS "Found pybind11 at ${pybind11_CONFIG}")
+    endif()
+else()
+    message(STATUS "Using src/deps/pybind11 by default")
+endif()
+
+
 #----- RapidJSON

 if(OTIO_FIND_RAPIDJSON)
diff --git src/deps/CMakeLists.txt src/deps/CMakeLists.txt
index d99dd79ac..bfda2778a 100644
--- src/deps/CMakeLists.txt
+++ src/deps/CMakeLists.txt
@@ -4,14 +4,21 @@
 #----- Other dependencies

 # detect if the submodules haven't been updated
-set(DEPS_SUBMODULES pybind11)
+if(NOT OTIO_FIND_PYBIND11 AND OTIO_PYTHON_INSTALL)
+    # pybind11 only needed when building Python bindings
+    set(DEPS_SUBMODULES ${DEPS_SUBMODULES} pybind11)
+endif()
+
+if(NOT OTIO_FIND_RAPIDJSON)
+    set(DEPS_SUBMODULES ${DEPS_SUBMODULES} rapidjson)
+endif()

 if(NOT OTIO_FIND_RAPIDJSON)
     set(DEPS_SUBMODULES ${DEPS_SUBMODULES} rapidjson)
 endif()

 foreach(submodule IN LISTS DEPS_SUBMODULES)
-    file(GLOB SUBMOD_CONTENTS ${submodule})
+    file(GLOB SUBMOD_CONTENTS "${submodule}/*")
     list(LENGTH SUBMOD_CONTENTS SUBMOD_CONTENT_LEN)
     if(SUBMOD_CONTENT_LEN EQUAL 0)
         message(
@@ -21,10 +28,13 @@
     endif()
 endforeach()

-if(OTIO_PYTHON_INSTALL)
+if(NOT OTIO_FIND_PYBIND11 AND OTIO_PYTHON_INSTALL)
+    # pybind11 only needed when building Python bindings
     add_subdirectory(pybind11)
 endif()

+# Don't try to build rapidjson, it won't, and since it's header-only, we don't need to built it.
+
 if(NOT OTIO_FIND_IMATH)
     # preserve BUILD_SHARED_LIBS options for this project, but set it off for Imath
     option(BUILD_SHARED_LIBS "Build shared libraries" ON)
diff --git src/opentime/CMakeLists.txt src/opentime/CMakeLists.txt
index d99dd79ac..bfda2778a 100644
--- src/opentime/CMakeLists.txt
+++ src/opentime/CMakeLists.txt
@@ -73,7 +73,7 @@
             RUNTIME DESTINATION "${OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR}")

     install(EXPORT OpenTimeTargets
-            DESTINATION "${OTIO_RESOLVED_CXX_INSTALL_DIR}/share/opentime"
+            DESTINATION "${OTIO_RESOLVED_CXX_INSTALL_DIR}/lib/cmake/opentime"
             NAMESPACE OTIO:: )

     include(CMakePackageConfigHelpers)
@@ -81,7 +81,7 @@
         ${CMAKE_CURRENT_SOURCE_DIR}/OpenTimeConfig.cmake.in
         ${CMAKE_CURRENT_BINARY_DIR}/OpenTimeConfig.cmake
         INSTALL_DESTINATION
-            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/share/opentime
+            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/lib/cmake/opentime
         NO_SET_AND_CHECK_MACRO
         NO_CHECK_REQUIRED_COMPONENTS_MACRO
     )
@@ -90,7 +90,7 @@
         FILES
             ${CMAKE_CURRENT_BINARY_DIR}/OpenTimeConfig.cmake
         DESTINATION
-            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/share/opentime
+            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/lib/cmake/opentime
     )

     install(
diff --git src/opentimelineio/CMakeLists.txt src/opentimelineio/CMakeLists.txt
index d99dd79ac..bfda2778a 100644
--- src/opentimelineio/CMakeLists.txt
+++ src/opentimelineio/CMakeLists.txt
@@ -151,7 +151,7 @@
            RUNTIME DESTINATION "${OTIO_RESOLVED_CXX_DYLIB_INSTALL_DIR}")

     install(EXPORT OpenTimelineIOTargets
-           DESTINATION "${OTIO_RESOLVED_CXX_INSTALL_DIR}/share/opentimelineio"
+           DESTINATION "${OTIO_RESOLVED_CXX_INSTALL_DIR}/lib/cmake/opentimelineio"
            NAMESPACE OTIO:: )

     include(CMakePackageConfigHelpers)
@@ -159,7 +159,7 @@
         ${CMAKE_CURRENT_SOURCE_DIR}/OpenTimelineIOConfig.cmake.in
         ${CMAKE_CURRENT_BINARY_DIR}/OpenTimelineIOConfig.cmake
         INSTALL_DESTINATION
-            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/share/opentimelineio
+            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/lib/cmake/opentimelineio
         NO_SET_AND_CHECK_MACRO
         NO_CHECK_REQUIRED_COMPONENTS_MACRO
     )
@@ -168,7 +168,7 @@
         FILES
             ${CMAKE_CURRENT_BINARY_DIR}/OpenTimelineIOConfig.cmake
         DESTINATION
-            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/share/opentimelineio
+            ${OTIO_RESOLVED_CXX_INSTALL_DIR}/lib/cmake/opentimelineio
     )

     install(
