--- cmake/modules/MaterialXConfig.cmake.in
+++ cmake/modules/MaterialXConfig.cmake.in
@@ -18,10 +18,10 @@
 # MATERIALX_RESOURCES_DIR    Path to MaterialX Resources (sample data, mtlx etc)

 set_and_check(MATERIALX_BASE_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@")
-set_and_check(MATERIALX_STDLIB_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/libraries")
+set_and_check(MATERIALX_STDLIB_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/@MATERIALX_INSTALL_STDLIB_PATH@")
 if(@MATERIALX_BUILD_PYTHON@ AND @MATERIALX_INSTALL_PYTHON@)
     set_and_check(MATERIALX_PYTHON_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/python")
 endif()
-set_and_check(MATERIALX_RESOURCES_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/resources")
+set_and_check(MATERIALX_RESOURCES_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/@MATERIALX_INSTALL_RESOURCES_PATH@")

 check_required_components(@CMAKE_PROJECT_NAME@)
--- resources/CMakeLists.txt
+++ resources/CMakeLists.txt
@@ -1,2 +1,2 @@
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
-        DESTINATION "resources" MESSAGE_NEVER)
+        DESTINATION "${MATERIALX_INSTALL_RESOURCES_PATH}" MESSAGE_NEVER)
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -73,6 +73,7 @@
 set(MATERIALX_INSTALL_INCLUDE_PATH "include" CACHE STRING "Install header include path (e.g. 'inc', 'include').")
 set(MATERIALX_INSTALL_LIB_PATH "lib" CACHE STRING "Install lib path (e.g. 'libs', 'lib').")
 set(MATERIALX_INSTALL_STDLIB_PATH "libraries" CACHE STRING "Install path for mtlx std libs (e.g. 'libraries').")
+set(MATERIALX_INSTALL_RESOURCES_PATH "resources" CACHE STRING "Install path for mtlx resources / test files (e.g. 'resources').")

 # Helpers for OSL validation
 set(MATERIALX_OSL_BINARY_OSLC "" CACHE FILEPATH "Full path to the OSL compiler binary.")
