--- cmake/modules/MaterialXConfig.cmake.in
+++ cmake/modules/MaterialXConfig.cmake.in
@@ -18,7 +18,7 @@
 # MATERIALX_RESOURCES_DIR    Path to MaterialX Resources (sample data, mtlx etc)

 set_and_check(MATERIALX_BASE_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@")
-set_and_check(MATERIALX_STDLIB_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/libraries")
+set_and_check(MATERIALX_STDLIB_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/share/MaterialX")
 if(@MATERIALX_BUILD_PYTHON@ AND @MATERIALX_INSTALL_PYTHON@)
     set_and_check(MATERIALX_PYTHON_DIR "@PACKAGE_CMAKE_INSTALL_PREFIX@/python")
 endif()
