diff --git cmake/OpenFX.cmake cmake/OpenFX.cmake
index d99dd79ac..bfda2778a 100644
--- cmake/OpenFX.cmake
+++ cmake/OpenFX.cmake
@@ -45,7 +45,10 @@
   set(BUNDLE_SIGNATURE "????")  # any value is OK here, ignored by modern MacOS
   set(PLUGIN_VERSION "1.0.0")

-  configure_file(${CMAKE_SOURCE_DIR}/Examples/Info.plist.in
+  # For the Conan test_package
+  set_target_properties(${TARGET} PROPERTIES EXAMPLES_DIR "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../Examples")
+
+  configure_file(${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../Examples/Info.plist.in
                  "${PLUGIN_INSTALLDIR}/${TARGET}.ofx.bundle/Contents/Info.plist")

   # Set symbol visibility hidden. Individual symbols are exposed via
