diff --git src/cmake/llvm_macros.cmake src/cmake/llvm_macros.cmake
index d99dd79ac..bfda2778a 100644
--- src/cmake/llvm_macros.cmake
+++ src/cmake/llvm_macros.cmake
@@ -35,8 +35,8 @@
         if (NOT LLVM_BC_GENERATOR)
             find_program (LLVM_BC_GENERATOR NAMES "clang++"
                         PATHS "${LLVM_DIRECTORY}/bin" "${LLVM_DIRECTORY}/tools/llvm"
-                        NO_CMAKE_PATH NO_DEFAULT_PATH NO_CMAKE_SYSTEM_PATH
-                        NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_ENVIRONMENT_PATH)
+                        NO_CMAKE_PATH NO_CMAKE_SYSTEM_PATH
+                        NO_CMAKE_ENVIRONMENT_PATH)
         endif ()

         if (NOT LLVM_BC_GENERATOR)
@@ -49,15 +49,15 @@
         if (NOT LLVM_AS_TOOL)
             find_program (LLVM_AS_TOOL NAMES "llvm-as"
                         PATHS "${LLVM_DIRECTORY}/bin" "${LLVM_DIRECTORY}/tools/llvm"
-                        NO_CMAKE_PATH NO_DEFAULT_PATH NO_CMAKE_SYSTEM_PATH
-                        NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_ENVIRONMENT_PATH)
+                        NO_CMAKE_PATH NO_CMAKE_SYSTEM_PATH
+                        NO_CMAKE_ENVIRONMENT_PATH)
         endif ()

         if (NOT LLVM_LINK_TOOL)
             find_program (LLVM_LINK_TOOL NAMES "llvm-link"
                     PATHS "${LLVM_DIRECTORY}/bin" "${LLVM_DIRECTORY}/tools/llvm"
-                    NO_CMAKE_PATH NO_DEFAULT_PATH NO_CMAKE_SYSTEM_PATH
-                    NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_ENVIRONMENT_PATH)
+                    NO_CMAKE_PATH NO_CMAKE_SYSTEM_PATH
+                    NO_CMAKE_ENVIRONMENT_PATH)
         endif ()


@@ -75,11 +75,11 @@
         #    endif ()
         #endif ()

-        list (TRANSFORM IMATH_INCLUDES PREPEND -I
+        list (TRANSFORM Imath_INCLUDE_DIRS PREPEND -I
             OUTPUT_VARIABLE ALL_IMATH_INCLUDES)
-        list (TRANSFORM OPENEXR_INCLUDES PREPEND -I
+        list (TRANSFORM OpenEXR_INCLUDE_DIRS PREPEND -I
             OUTPUT_VARIABLE ALL_OPENEXR_INCLUDES)
-        list (TRANSFORM OpenImageIO_INCLUDES PREPEND -I
+        list (TRANSFORM OpenImageIO_INCLUDE_DIRS PREPEND -I
             OUTPUT_VARIABLE ALL_OpenImageIO_INCLUDES)

         if (${LLVM_VERSION} VERSION_GREATER_EQUAL 15.0)
--- src/cmake/cuda_macros.cmake
+++ src/cmake/cuda_macros.cmake
@@ -14,11 +14,11 @@
     file ( GLOB cuda_headers "${cuda_src_dir}/*.h" )
     list (APPEND cuda_headers ${extra_headers})

-    list (TRANSFORM IMATH_INCLUDES PREPEND -I
+    list (TRANSFORM Imath_INCLUDE_DIRS PREPEND -I
           OUTPUT_VARIABLE ALL_IMATH_INCLUDES)
-    list (TRANSFORM OPENEXR_INCLUDES PREPEND -I
+    list (TRANSFORM OpenEXR_INCLUDE_DIRS PREPEND -I
           OUTPUT_VARIABLE ALL_OPENEXR_INCLUDES)
-    list (TRANSFORM OpenImageIO_INCLUDES PREPEND -I
+    list (TRANSFORM OpenImageIO_INCLUDE_DIRS PREPEND -I
           OUTPUT_VARIABLE ALL_OpenImageIO_INCLUDES)

     if (${LLVM_VERSION} VERSION_GREATER_EQUAL 15.0)
@@ -71,7 +71,7 @@

     # Setup the bitcode generator
     if (NOT LLVM_BC_GENERATOR)
-        FIND_PROGRAM(LLVM_BC_GENERATOR NAMES "clang++" PATHS "${LLVM_DIRECTORY}/bin" NO_DEFAULT_PATH NO_CMAKE_SYSTEM_PATH NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_ENVIRONMENT_PATH NO_CMAKE_PATH)
+        FIND_PROGRAM(LLVM_BC_GENERATOR NAMES "clang++" PATHS "${LLVM_DIRECTORY}/bin" NO_CMAKE_PATH NO_CMAKE_SYSTEM_PATH NO_CMAKE_ENVIRONMENT_PATH)
     endif ()
     # If that didn't work, look anywhere
     if (NOT LLVM_BC_GENERATOR)
@@ -89,15 +89,15 @@
     if (NOT LLVM_AS_TOOL)
       find_program (LLVM_AS_TOOL NAMES "llvm-as"
                 PATHS "${LLVM_DIRECTORY}/bin" "${LLVM_DIRECTORY}/tools/llvm"
-                NO_CMAKE_PATH NO_DEFAULT_PATH NO_CMAKE_SYSTEM_PATH
-                NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_ENVIRONMENT_PATH)
+                NO_CMAKE_PATH NO_CMAKE_SYSTEM_PATH
+                NO_CMAKE_ENVIRONMENT_PATH)
     endif ()

     if (NOT LLVM_LINK_TOOL)
         find_program (LLVM_LINK_TOOL NAMES "llvm-link"
                 PATHS "${LLVM_DIRECTORY}/bin" "${LLVM_DIRECTORY}/tools/llvm"
-                NO_CMAKE_PATH NO_DEFAULT_PATH NO_CMAKE_SYSTEM_PATH
-                NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_ENVIRONMENT_PATH)
+                NO_CMAKE_PATH NO_CMAKE_SYSTEM_PATH
+                NO_CMAKE_ENVIRONMENT_PATH)
     endif ()

     if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
@@ -110,11 +110,11 @@
         set (CLANG_MSVC_FIX "${CLANG_MSVC_FIX} -Wno-ignored-attributes -Wno-unknown-attributes")
     endif ()

-    list (TRANSFORM IMATH_INCLUDES PREPEND -I
+    list (TRANSFORM Imath_INCLUDE_DIRS PREPEND -I
           OUTPUT_VARIABLE ALL_IMATH_INCLUDES)
-    list (TRANSFORM OPENEXR_INCLUDES PREPEND -I
+    list (TRANSFORM OpenEXR_INCLUDE_DIRS PREPEND -I
           OUTPUT_VARIABLE ALL_OPENEXR_INCLUDES)
-    list (TRANSFORM OpenImageIO_INCLUDES PREPEND -I
+    list (TRANSFORM OpenImageIO_INCLUDE_DIRS PREPEND -I
           OUTPUT_VARIABLE ALL_OpenImageIO_INCLUDES)

     add_custom_command (OUTPUT ${bc_cuda}
