diff --git src/testshade/CMakeLists.txt src/testshade/CMakeLists.txt
index d99dd79ac..bfda2778a 100644
--- src/testshade/CMakeLists.txt
+++ src/testshade/CMakeLists.txt
@@ -68,9 +68,9 @@
 set(include_dirs ${CMAKE_CURRENT_SOURCE_DIR})
 list(APPEND include_dirs ${CMAKE_SOURCE_DIR}/src/include)
 list(APPEND include_dirs ${CMAKE_BINARY_DIR}/include)
-list(APPEND include_dirs ${IMATH_INCLUDES})
-list(APPEND include_dirs ${OPENEXR_INCLUDES})
-list(APPEND include_dirs ${OpenImageIO_INCLUDES})
+list(APPEND include_dirs ${Imath_INCLUDE_DIRS})
+list(APPEND include_dirs ${OpenEXR_INCLUDE_DIRS})
+list(APPEND include_dirs ${OpenImageIO_INCLUDE_DIRS})
 list(APPEND include_dirs ${Boost_INCLUDE_DIRS})

 EMBED_LLVM_BITCODE_IN_CPP ( "${rs_srcs}" "_host" "testshade_llvm_compiled_rs" testshade_srcs "-DOSL_HOST_RS_BITCODE=1" "${include_dirs}")
@@ -80,6 +80,9 @@
 target_link_libraries (testshade
                        PRIVATE
                            oslexec oslquery)
+if (OSL_USE_OPTIX)
+    add_dependencies(testshade testshade_ptx)
+endif ()

 install (TARGETS testshade RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} )

@@ -96,6 +99,7 @@
 if (NOT CODECOV)
     # The 'libtestshade' library
     add_library ( "libtestshade" ${testshade_srcs} )
+    add_dependencies(libtestshade testshade)

     set_target_properties (libtestshade
                            PROPERTIES
