diff --git src/testshade/CMakeLists.txt src/testshade/CMakeLists.txt
index d99dd79ac..bfda2778a 100644
--- src/testshade/CMakeLists.txt
+++ src/testshade/CMakeLists.txt
@@ -80,6 +80,9 @@
 target_link_libraries (testshade
                        PRIVATE
                            oslexec oslquery)
+if (OSL_USE_OPTIX)
+    add_dependencies(testshade testshade_ptx)
+endif ()

 install (TARGETS testshade RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} )

@@ -96,6 +99,7 @@
 if (NOT CODECOV)
     # The 'libtestshade' library
     add_library ( "libtestshade" ${testshade_srcs} )
+    add_dependencies(libtestshade testshade)

     set_target_properties (libtestshade
                            PROPERTIES
